domain/learningandworkprogress/src/test/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/service/InductionServiceTest.kt

```diff
@@ -41,7 +42,7 @@ class InductionServiceTest {
   @Nested
   inner class CreateInduction {
     @Test
-    fun `should create induction`() {
+    fun `should create induction`() = runTest {
```

Comment:
All the tests that invoke a `suspend` function now need to be wrapped in `runTest` which provides the coroutine scope to the function